nntrainer/graph/network_graph.cpp

```diff
@@ -140,7 +139,7 @@ int NetworkGraph::realizeMultiInputType(Layer &current) {
     return ML_ERROR_NONE;
 
   if (current.num_inputs > 1) {
-    std::shared_ptr<Layer> layer = nntrainer::createLayer("concat");
+    std::shared_ptr<Layer> layer = nntrainer::createLayer("addition");
```

Comment:
Could you check if this has to be addition only? I think this should be either concat or addition.

Reply:
Will recomment on the latest patch